_posts/news/2016-10-17-announcing-scalajs-0.6.13.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: post
+title: Announcing Scala.js 0.6.13
+category: news
+tags: [releases]
+permalink: /news/2016/09/01/announcing-scalajs-0.6.13/
+---
+
+
+We are excited to announce the release of Scala.js 0.6.13!
+
+This release one particularly anticipated feature: the ability to generate CommonJS modules with Scala.js!
+It also standardizes on Node.js as the default runner for all sbt projects (with `sbt run` and `sbt test`), which constitutes a breaking change for builds.
+Read on for more details!
+
+<!--more-->
+
+## Getting started
+
+If you are new to Scala.js, head over to
+[the tutorial]({{ BASE_PATH }}/tutorial/).
+
+## Release notes
+
+As a minor release, 0.6.13 is backward source and binary compatible with previous releases in the 0.6.x series, although *build* definitions are not.
+Libraries compiled with earlier versions can be used with 0.6.13 without change.
+However, it is not forward compatible: libraries compiled with 0.6.13 cannot be used by projects using 0.6.{0-12}.
+
+Please report any issues [on GitHub](https://github.com/scala-js/scala-js/issues).
+
+## Breaking changes
+
+This release changes the default JavaScript interpreters used to perform `sbt run` and `sbt test`.
+Until 0.6.12 (and since 0.6.6), the defaults were:
+
+* Rhino by default
+* Node.js with `scalaJSUseRhino in Global := false`
+* PhantomJS with `scalaJSUseRhino in Global := false` and `jsDependencies += RuntimeDOM`
+
+The new defaults are the following:
+
+* Node.js by default
+* Node.js + [`jsdom`](https://github.com/tmpvar/jsdom) with `jsDependencies += RuntimeDOM`
+
+Note that Node.js and jsdom need to be installed separately on your system.
+
+* [Download Node.js here](https://nodejs.org/en/download/)
+* Install jsdom with `npm install jsdom` (either in your project's root directory, or [globally](https://docs.npmjs.com/getting-started/installing-npm-packages-globally))
+
+We decided to standardize on Node.js for all command-line executions by default because it is always the best alternative.
+Rhino is extremely slow.
+Using by default, although working out-of-the-box, caused a number of users to stick to it and discovering months or years later that their tests could run 10x-100x faster by switching to Node.js.
```

Comment:
Consider rephrasing to: Although using it by default works out-of-the-box, it caused a number of users to stick to it and discover months [...] faster just by switching to Node.js.